diff --git a/CMakeLists.txt b/CMakeLists.txt
index a5a1619..747f14b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -35,7 +35,7 @@ INCLUDE_DIRECTORIES( ${MOAB_INCLUDE_DIRS} )
 # Embree
 # this series of calls is needed to support the way that Embree indicates
 # compatible versions in 3 and 4 (version 4 does it correctly, but 3 does not)
-FIND_PACKAGE(embree 3.6.1 HINTS ${EMBREE_DIR} QUIET)
+find_package(embree MODULE)
 if (NOT ${embree_FOUND})
 FIND_PACKAGE (embree REQUIRED HINTS ${EMBREE_DIR})
 endif()
